swagger.json

```diff
@@ -502,6 +502,75 @@
         }
       }
     },
+    "/token": {
+      "post": {
+        "summary": "Create a token",
+        "description": "Adds a new API token",
+        "parameters": [{
+            "name": "X-API-KEY",
+            "in": "header",
+            "required": true
+          }],
+        "requestBody": {
+          "description": "username",
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "username": {
+                    "description": "Name of person generating token",
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": {
+          "ApiKeyAuth": []
+        },
+        "responses": {
+          "201": {
+            "description": "Token created",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "X-API-KEY": {
+                      "description": "API token",
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "409": {
+            "description": "Submitted parameters do not match parameters for an existing user",
```

Comment:
Is this status being returned somewhere? What user is the description referring to? If the user making the request doesn't exist, then authentication will fail and return 401. The user for whom the token is being created doesn't need to exist